wurst/_items/Blowgun.wurst

```diff
@@ -158,10 +158,8 @@ class Blowpipe
             updateTooltip()
 
     function loadAmmo(unit owner, item itm)
-        int charges = itm.getCharges()
         //Treat non-charge items as 1 charge
-        if charges == 0
-            charges = 1
+        int charges = max(itm.getCharges(),1)
```

Comment:
You're missing a space. Did you try using `or`?

Reply:
yes